insights/specs/default.py

```diff
@@ -662,4 +662,5 @@ def sap_sid_name(broker):
     yum_repos_d = glob_file("/etc/yum.repos.d/*")
     zipl_conf = simple_file("/etc/zipl.conf")
     rpm_format = format_rpm()
+    initscript = glob_file("/etc/rc.d/init.d/*")
```

Comment:
Please insert this spec per the alpha order. (installed_rpm is an exception since it depends on "rpm_format")

Reply:
@xiangce - Done,  please check